src/components/VMCreator.jsx

```diff
@@ -23,6 +24,8 @@ class VMCreator extends React.Component {
       progressUploadXML: 0,
       progressUploadQCOW: 0,
       isVMCreated: null,
+      isLiveMigrationEnabled: false,
+      migrationUser: 'root',
```

Comment:
Should I put root as default live migration user or something else or nothing at all ?

Reply:
TBC, but there maybe a security issue to let the root user to do the migration (root user is banned by default in Cockpit). AFAK, I would let, if possible the field blank